subsys/debug/Kconfig

```diff
@@ -350,6 +350,12 @@ config TRACING_CTF_BOTTOM_POSIX
 	  Enable POSIX backend for CTF tracing. It will output the CTF stream to a
 	  file using fwrite.
 
+config TRACING_CTF_BOTTOM_GENERIC
```

Comment:
I still think it should be called "buffered" rather than "generic", as that is what it is.